VERSIONING.md

```diff
@@ -72,7 +72,7 @@ respect to semantic versioning.
 | Language | Minimum Version                                                                                                                                                                                                                  | Applicability                                           | Semconv Notes                                                                                                                                                                      |
 |----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Java     | 8+                                                                                                                                                                                                                               | All artifacts, unless otherwise noted                   | Changing requires major version bump.                                                                                                                                              |
-| Android  | 21+ (NOTE: [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) is required. We stay up to date with the latest version of [desugar_jdk_libs](https://github.com/google/desugar_jdk_libs).) | Artifacts using `otel.animalsniffer-conventions` plugin | Kept in sync with minimum requirements for [Google Play services](https://developers.google.com/android/guides/setup). Subject to change in minor version.                         |
+| Android  | 23+ (NOTE: [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) is required. We stay up to date with the latest version of [desugar_jdk_libs](https://github.com/google/desugar_jdk_libs).) | Artifacts using `otel.animalsniffer-conventions` plugin | Kept in sync with minimum requirements for [Google Play services](https://developers.google.com/android/guides/setup). Subject to change in minor version.                         |
```

Comment:
This change is to conform to our policy:

> Kept in sync with minimum requirements for [Google Play services](https://developers.google.com/android/guides/setup).

The minimum version has been updated from 21 to 23.